### ISSUE TITLE
Screwdrivers to the eye instant stun, blind, disarm & don't respect eye protection or hit-chance.

### ISSUE THREAD

**Issue by FenodyreeAv** [PolarisSS13/Polaris#6332]
#### Brief description of the issue
Attacking someone in the eyes with a screwdriver doesn't check their glasses, causes an instant knockdown, blindness and can permastun someone to death.

#### What you expected to happen
Eye damage, with a low hit chance, deflected by eye-wear.

#### What actually happened
Peramstun to RIP while wearing goggles.

#### Steps to reproduce
Stab a dude in the eye with a screwdriver. Preferably in game.

#### Additional info:
 - **Server Revision**: Found using the "Show Server Revision" verb under the OOC tab.
 - **Anything else you may wish to add** (Location if it's a mapping issue, etc)

**Comment by DasIrrlicht** [comment]
Also applys for Pocket Sand.

**Comment by MagmaRam** [comment]
Can't reproduce the protection part in testing. Science goggles, mesons, welding masks, and other things that include eyes in body_parts_covered do protect against eye stabs. Notably, normal glasses do NOT include this, which is something that is apparently intentional.

**Comment by FenodyreeAv** [comment]
Ah, I was wearing material scanners at the time.

**Comment by MagmaRam** [comment]
Oh, piss, I'm going to have to go through all the head/mask/glasses items and check them for proper coverage flags after all, aren't I?

**Comment by DasIrrlicht** [comment]
Yup. Also includes Shroud-Helmets and Veils for Pocketsand. + Googles, hit past three layers of protection.
